### ISSUE TITLE
GNOME decorations hover should not use a gradient

### ISSUE THREAD

**Issue by gsbhasin123** [kupiqu/SierraBreezeEnhanced#72]
The latest Adwaita version does not use a gradient when you hover over a button.

**Comment by kupiqu** [comment]
I heard about that, it seems this #21 does not apply anymore :(

I think there is some screenshot in some other issue, I'll try to find it

**Comment by kupiqu** [comment]
Oh, it's actually in #65

**Comment by gsbhasin123** [comment]
Ah ok ;)

**Comment by gsbhasin123** [comment]
Yup
![image](https://user-images.githubusercontent.com/48339289/78398442-f21a0080-75a7-11ea-976e-2581f3c749cc.png)
Adwaita was updated for a more flat look :D

**Comment by kupiqu** [comment]
I guess it makes sense to change it if Gnome/Adwaita style users prefer it this way. Would you like to give it a try? It shouldn't be too difficult to change it

**Comment by gsbhasin123** [comment]
Sure! Where would I find the relevant files?

**Comment by kupiqu** [comment]
It's all in breezebutton.cpp

**Comment by gsbhasin123** [comment]
Hm, I don't know C++. I'll take a look and see what I can do though

**Comment by kupiqu** [comment]
> Hm, I don't know C++. I'll take a look and see what I can do though

I don't think you need much C++ knowledge for this purpose, give it a try :-)

**Comment by gsbhasin123** [comment]
I'm assuming you can just remove all the gradient references?
And then paint the standard color object instead of a gradient object?

**Comment by kupiqu** [comment]
Yes, just that. I recommend that you make changes one by one and check it on the fly.

And then you can consider whether buttons should be surrounded by circles or squares according to Adwaita...

**Comment by kupiqu** [comment]
Also, wondering whether we should keep previous Gnome theme for those who like it, hmm

**Comment by gsbhasin123** [comment]
Why not call this one adwaita and the old one 'old adwaita'?

Oh, ok

**Comment by kupiqu** [comment]
> Why not call this one adwaita and the old one 'old adwaita'?
> 
> Oh, ok

Sure, I am all for it, didn't want to give you too much trouble ;-)

**Comment by gsbhasin123** [comment]
:P 
I have a school assignment to do;I'll give it a shot after!

**Comment by kupiqu** [comment]
Sure, no hurries

**Comment by gsbhasin123** [comment]
got sidetracked. I cannot figure it out :(

**Comment by kupiqu** [comment]
Ok, no problem

**Comment by kupiqu** [comment]
My aim is that SBE is maintained by the community. This shouldn't be hard to fix. I hope someone in the community cares about SBE and submits a pull request about it... I could help if somebody wants to try but is lost where to start.
